docs/docs/migration/v0.25.0.md

```diff
@@ -1,7 +1,8 @@
 ---
 sidebar_position: 0
 title: v0.25.0
-description: For chains that were scaffolded with Ignite CLI versions lower than v0.25.0. changes are required to use Ignite CLI v0.24.0.
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.25.0. changes are required to use
+Ignite CLI v0.25.0.
```

Comment:
This is a little too aggressive, it breaks the yaml :P 

I'll fix those in the cosmoscmd migration PR.